clang/lib/Sema/SemaTemplateDeduction.cpp

```diff
@@ -2552,7 +2553,7 @@ DeduceTemplateArguments(Sema &S, TemplateParameterList *TemplateParams,
                         ArrayRef<TemplateArgument> As,
                         TemplateDeductionInfo &Info,
                         SmallVectorImpl<DeducedTemplateArgument> &Deduced,
-                        bool NumberOfArgumentsMustMatch) {
+                        bool NumberOfArgumentsMustMatch, bool Swapped) {
```

Comment:
I think this needs a comment.
`Swapped` should be defaulted and have a better name.
Maybe `ForPartialOrderingOfDeducedTemplateTemplateArgument`